addons/web/static/src/webclient/actions/reports/report.scss

```diff
@@ -119,3 +119,16 @@ ol {
     -webkit-box-flex: 1 !important;
     width: 100% !important;
 }
+
+// Boostrap 5 introduces variable paddings for container which wkhtmltopdf doesn't seem to process, so we restore Boostrap 4's paddings for PDFs
+.container {
+    padding-right: $container-padding-x;
+    padding-left: $container-padding-x;
+}
+
+// Reintroduces table-borderless from Boostrap 4 as the one from Boostrap 5 still has its borders visible in PDFs 
```

Comment:
Adapt this comment as it don't came from Bootstrap 4 anymore.

Reply:
Indeed, rewrote the comment.

Reply:
Also you have the "same" comment in the commit and the PR

Reply:
Right. Edited the commit message and the PR description.